lib/spack/spack/test/llnl/util/filesystem.py

```diff
@@ -162,6 +178,15 @@ def test_symlinks_false(self, stage):
             assert os.path.exists('dest/2')
             assert not os.path.islink('dest/2')
 
+    def test_glob_src(self, stage):
+        """Test using a glob as the source."""
+
+        with fs.working_dir(str(stage)):
+            fs.copy_tree('source/g/*', 'dest')
+
+            assert os.path.exists('dest/8')
+            assert os.path.exists('dest/9')
```

Comment:
Can you make the (copy|install)_tree glob src tests use a source with some internal structure?